region/crateria/east.json

```diff
@@ -1482,6 +1482,24 @@
                   ],
                   "devNote": "Could use a failure definition?"
                 },
+                {
+                  "name": "West Ocean Bug Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "canNonTrivialIceClip"
+                  ],
+                  "note": [
+                    "Freeze a bug a three tiles to the right of the morph tunnel entrance, directly under the tile where ceiling is higher.",
```

Comment:
I'd like it pointed out that it looks like a 3 tile high ceiling.  Less important would be to say its one solid tile and two slopes.

Reply:
Done.